lib/active_admin/views/pages/show.rb

```diff
@@ -25,7 +25,12 @@ def main_content
         end
 
         def attributes_table(*args, &block)
-          table_title = ActiveAdmin::Localizers.resource(active_admin_config).t(:details)
+          opts = args.extract_options!
+          table_title = if opts.has_key?( :title )
```

Comment:
Yes, I'm not asking for any changes in this PR, just taking the chance to discuss the style we like. :)